src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const NEW_STATE = { ...state };
+  const STATES = [];
+
+  for (const ACTION of actions) {
```

Comment:
The variable 'ACTION' should be in camelCase to follow JavaScript naming conventions, such as 'action'.